typegate/tests/runtimes/deno/deno_test.ts

```diff
@@ -155,3 +156,36 @@ Meta.test("Deno runtime: reloading", async (t) => {
       .on(v2);
   });
 });
+
+Meta.test("Deno runtime: infinite loop or similar", async (t) => {
+  const e = await t.engine("runtimes/deno/deno.py");
+
+  await t.should("safely fail upon stack oferflow", async () => {
+    await gql`
+      query {
+        stackOverflow(enable: true)
+      }
+    `
+      .expectErrorContains("Maximum call stack size exceeded")
+      .on(e);
+  });
+
+  // hack for avoiding leaking ops in test context
+  // This    --- OpOpen ---- [TestOpen --- OpClose --- TestClose]
+  // becomes --- OpOpen ---- [TestOpen --- TestClose] --- OpClose
+  const configValue = config.timer_destroy_ressources;
+  config.timer_destroy_ressources = false;
+  await t.should(
+    "safely fail upon an infinite loop",
+    async () => {
+      await gql`
+          query {
+            infiniteLoop(enable: true)
+          }
+        `
+        .expectErrorContains("timeout exceeded")
+        .on(e);
+    },
+  );
+  config.timer_destroy_ressources = configValue;
```

Comment:
Test that request can still continue afterwards?

Reply:
This was an `async ops` hack. This test seems to break the ci anyways, it makes test-typegate running indefinitely  (vs local). Still working on how to tackle it properly.